### PR TITLE
Fixes #23259 - Fix resource switcher for breadcrumbs

### PR DESCRIPTION
**Pull request by xprazak2**
I am not entirely sure what we should display in breadcrumbs for arf report, I left the hostname there.

### CODE REVIEW

**Comment by ares** [comment]
ping @ohadlevy @amirfefer @sharvit - is this the way all plugins should be fixed? or is there some fix in core possible? what is the recommendation for arf report and how we can potentially disable it on that page, if it does not make sense there?

**Comment by xprazak2** [comment]
Fixing rubocop offences.

**Comment by sharvit** [comment]
@ares, I hope it will help to make it more clear:
https://community.theforeman.org/t/two-pane-is-deprecated-how-to-switch-to-the-new-breadcrumbbar/8902

File: app/views/policies/edit.html.erb [hunk 1]
```diff
@@ -1,2 +1,13 @@
 <% title _("Edit Compliance Policy") %>
+<%= breadcrumbs(:resource_url => api_compliance_policies_path,
```
[the remark this answers is not in the document]
Comment (ares) [reply app/views/policies/edit.html.erb]: just realized that the workaround wouldn't work, it seems the breadcrumbs helper hardcodes using `controller_name` and I don't see how to pass the namespace (ideally the helper should determine it)

**Pull request merged**